### PR TITLE
Issue #519: remove from all properties files InputCyclomaticComplexity.java

### LINKED ISSUE #519: remove from all properties files InputCyclomaticComplexityRecords.java
as https://github.com/checkstyle/checkstyle/pull/8913
merged we need to remove from all properties files reference to exclude InputCyclomaticComplexityRecords.java
as it will be not a problem any more.

### PR DESCRIPTION
**Pull request by anhminhtran235**
Fixes #519

### CODE REVIEW

File: checkstyle-tester/projects-for-travis.properties [hunk 1]
```diff
@@ -5,5 +5,4 @@
 guava|git|https://github.com/google/guava|v28.2||
 # Few projects that delivers set of unusual Java constructions that shall be correctly handled by AST visitor
 # 'InputAllEscapedUnicodeCharacters' must be skipped because it is too big and slows down JXR
-# InputCyclomaticComplexityRecords.java - bug in checkstyle
 #checkstyle|git|https://github.com/checkstyle/checkstyle.git|master|**/.ci-temp/**/*,**/resources-noncompilable/**/asttreestringprinter/**/*,**/resources-noncompilable/**/filefilters/**/*,**/resources-noncompilable/**/main/**/*,**/resources-noncompilable/**/suppressionsstringprinter/**/*,**/resources-noncompilable/**/gui/**/*,**/resources-noncompilable/**/javadocpropertiesgenerator/**/*,src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/javaparser/InputJavaParser.java,**/InputAllEscapedUnicodeCharacters.java,**/resources-noncompilable/**/javaparser/InputJavaParser.java,**/resources-noncompilable/**/checks/metrics/cyclomaticcomplexity/InputCyclomaticComplexityRecords.java,**/resources-noncompilable/**/checks/imports/unusedimports/InputUnusedImportsSingleWordPackage.java
```
Comment (romani) [inline checkstyle-tester/projects-for-travis.properties:8]: Remove this file reference from this line, at the end of long line.

Please do this in all other times

**Review by romani — CHANGES_REQUESTED** [review]
Items:

File: checkstyle-tester/projects-for-travis.properties (hunk 1 above)
Comment (anhminhtran235) [reply checkstyle-tester/projects-for-travis.properties]: Fixed

**Review by romani — APPROVED** [review]
Ok to merge after fix in main project

**Comment by anhminhtran235** [comment]
@romani Should we merge this PR now?

**Review by romani — APPROVED** [review]

**Pull request merged**